templates/summary.html

```diff
@@ -16,15 +16,36 @@ <h1>{{ _("Check your answers and submit") }}</h1>
       <strong>{{ _("Please submit this survey to complete it") }}</strong>
       <p>{{ _("Check your answers before submitting") }}</p>
     {% endcall %}
-  {% elif content.summary.summary_type in ('SectionSummary') %}
-    {% set group = content.summary.groups %}
-
-    <h1>{{content.summary.title}}</h1>
-      {% call onsPanel({
-        "classes": "print__hidden u-mb-l"
-      }) %}
-        <p>{{ _("Check your answers before continuing to the next section") }}</p>
-      {% endcall %}
+  {% elif content.summary.summary_type == 'SectionSummary' %}
```

Comment:
There is a `sectionsummary.html` that we should make use of rather than the nested elif logic in this template.